CUSTOMISING.md

```diff
@@ -0,0 +1,107 @@
+
+# Customisation
+
+## Customizing DAMAP backend
+
+To migrate with the base, you need to fetch updates with upstream, following Semantic Versioning guidelines:
+- **Major:** Big changes (first value).
+- **Minor:** Additional changes (second value).
+For more details on Semantic Versioning, see [semver.org](https://semver.org/).
+
+In the `pom.xml`, update the dependency version as needed to match the new release of the base backend, for example:
+
+```xml
+  <!-- Base image -->
+  <dependency>
+    <groupId>org.damap</groupId>
+    <artifactId>base</artifactId>
+    <version>4.0.2</version>
+  </dependency>
+```
+
+You can implement your institutional backend by creating a new project with [quarkus](https://quarkus.io/).
+
+Download the damap-backend project and build it using:
+
+```shell
+mvn clean install -Dquarkus-profile=dev
+```
+
+After compiling the backend, you'll have a jar file in the target folder (e.g., `damap-backend-1.2.0.jar`).
+
+To include this in your institutional project, add the following dependency to your `pom.xml`:
+
+```xml
+<dependency>
+  <groupId>org.damap</groupId>
+  <artifactId>base</artifactId>
+  <type>jar</type>
+  <version>1.2.0-SNAPSHOT</version>
+</dependency>
+```
+
+Ensure the jar is either available locally or through your institutional Nexus.
+
+You can run the project using:
+
+```shell
+mvn compile quarkus:dev
+```
+

```

Comment:
It is usually the one or the other option. So one would choose one of two options
- only specify the DAMAP version as a dependency and it gets pulled from maven central automatically (https://central.sonatype.com/artifact/org.damap/base) (our approach)
- download the git repository, compile it locally and then specify it as a dependency. Then it should use the local one over the one from maven